Source/Aptabase/Private/Aptabase.cpp

```diff
@@ -40,4 +49,14 @@ void FAptabaseModule::OnApplicationShutdown()
 	AnalyticsProvider.Reset();
 }
 
+#ifdef WITH_EDITOR
+void FAptabaseModule::OnEndPIE(bool bIsSimulating)
+{
+	if (AnalyticsProvider.IsValid())
+	{
+		AnalyticsProvider->EndSession();
```

Comment:
This should automatically end the session if the user forgets to do so manually when play in editor.

If this doesn't happen, the `bHasActiveSession` might remain set across different sessions.